3주차/[Lv.2]_튜플/이상훈.js

```diff
@@ -0,0 +1,16 @@
+function solution(s) {
+  const answer = [];
+  const visitied = new Set();
+  s = JSON.parse(s.replaceAll('{', '[').replaceAll('}', ']')).sort((a, b) => a.length - b.length);
```

Comment:
```suggestion
  s = JSON.parse(s.replace(/{|}/g, (match) => match === '{' ? '[' : ']').sort((a, b) => a.length - b.length);
```
혹시 `replace()`함수의 두번째 인자로 [function](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/String/replace#%EB%A7%A4%EA%B0%9C%EB%B3%80%EC%88%98%EA%B0%80_function%EC%9C%BC%EB%A1%9C_%EC%A7%80%EC%A0%95%EB%90%98%EC%97%88%EC%9D%84_%EB%95%8C)을 받을 수 있는 거 아시나요? 🤔
참고하시면 좋을 것 같습니다!!